boms/integration-tests/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud</groupId>
+  <artifactId>integration-tests</artifactId>
+  <version>1.2.0-SNAPSHOT</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud.tools</groupId>
+      <artifactId>dependencies</artifactId>
+      <version>0.2.1-SNAPSHOT</version>
```

Comment:
Does this version get updated when `prepare_release.sh` runs in the root directory?
- If yes, no problem.
- If no, how about adding the parent pom as below?

```
  <parent>
    <groupId>com.google.cloud.tools</groupId>
    <artifactId>dependencies-parent</artifactId>
    <version>0.2.1-SNAPSHOT</version>
  </parent>
...
<dependencies>
    <dependency>
      <groupId>${project.groupId}</groupId>
      <artifactId>dependencies</artifactId>
      <version>${project.version}</version>
    </dependency>
   ...
```

Reply:
Yes, it does. I just tested it:

```
 <groupId>com.google.cloud</groupId>
  <artifactId>integration-tests</artifactId>
  <version>1.2.0-SNAPSHOT</version>

  <properties>
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    <maven.compiler.source>1.8</maven.compiler.source>
    <maven.compiler.target>1.8</maven.compiler.target>
  </properties>

  <dependencies>
    <dependency>
      <groupId>com.google.cloud.tools</groupId>
      <artifactId>dependencies</artifactId>
      <version>5.6.8-SNAPSHOT</version>
    </dependency>
    <dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
      <version>4.12</version>
      <scope>test</scope>
    </dependency>
```

Reply:
Yes, it does.